src/wp-load.php

```diff
@@ -21,6 +21,12 @@
 	define( 'ABSPATH', __DIR__ . '/' );
 }
 
+if ( ! defined( 'WPINC' ) ) {
+	define( 'WPINC', 'wp-includes' );
+}
+
+require_once ABSPATH . 'wp-includes/class-wp-autoload.php';
```

Comment:
Shouldn't this be:

```php
require_once ABSPATH . WPINC . '/class-wp-autoload.php';
```

Or else what's the point of moving `define( 'WPINC', 'wp-includes' );` above this line?

Reply:
`WPINC` is not used by the autoloader itself, but is used by other files.
It is a constast that cannot be overriden and I've seen a lot of people proposing we stop using it entirely since all it does is save us from typing 6 characters, and introduces additional string concatenation :shrug: 

Sure, if we feel it's necessary we can use the const, but that doesn't really change anything :+1:

Reply:
Personally I'd change it for consistency, but I kate you point.

Reply:
Note: the double concatenation will probably be optimized away in recent versions of PHP and not happen in the actual byte code.